src/app/components/client/ExposureCard.tsx

```diff
@@ -114,12 +113,125 @@ export const ExposureCard = (props: ExposureCardProps) => {
+    if (exposureItem.relatives.length > 0) {
+      exposureCategoriesArray.push(
+        <ScannedExposureCategory
+          key="relatives"
+          icon={<MultipleUsersIcon alt="" width="13" height="13" />}
+          exposureCategoryLabel={l10n.getString("exposure-card-family-members")}
+          num={exposureItem.relatives.length}
+        />
+      );
+    }
+    if (exposureItem.phones.length > 0) {
+      exposureCategoriesArray.push(
+        <ScannedExposureCategory
+          key="phones"
+          icon={<PhoneIcon alt="" width="13" height="13" />}
+          exposureCategoryLabel={l10n.getString("exposure-card-phone-number")}
+          num={exposureItem.phones.length}
+        />
+      );
+    }
+    if (exposureItem.emails.length > 0) {
+      exposureCategoriesArray.push(
+        <ScannedExposureCategory
+          key="emails"
+          icon={<EmailIcon alt="" width="13" height="13" />}
+          exposureCategoryLabel={l10n.getString("exposure-card-email")}
+          num={exposureItem.emails.length}
+        />
+      );
+    }
+    if (exposureItem.addresses.length > 0) {
+      exposureCategoriesArray.push(
+        <ScannedExposureCategory
+          key="addresses"
+          icon={<LocationPinIcon alt="" width="13" height="13" />}
+          exposureCategoryLabel={l10n.getString("exposure-card-address")}
+          num={exposureItem.addresses.length}
+        />
+      );
+    } else {
+      // "Other" item when none of the conditions above are met
+      exposureCategoriesArray.push(
+        <ScannedExposureCategory
+          key="other"
+          icon={<QuestionMarkCircle alt="" width="13" height="13" />}
+          exposureCategoryLabel={l10n.getString("exposure-card-other")}
+          num={0}
+        />
+      );
+    }
+  }
+
+  // Breach Categories
+  else {
+    exposureItem.DataClasses.map((item) => {
+      if (item === "email-addresses") {
+        exposureCategoriesArray.push(
+          <BreachExposureCategory
+            key={item}
+            icon={<EmailIcon alt="" width="13" height="13" />}
+            exposureCategoryLabel="Email"
+          />
+        );
+      } else if (item === "passwords") {
+        exposureCategoriesArray.push(
+          <BreachExposureCategory
+            key={item}
+            icon={<PasswordIcon alt="" width="13" height="13" />}
+            exposureCategoryLabel={l10n.getString("exposure-card-password")}
+          />
+        );
+      } else if (item === "phone-numbers") {
+        exposureCategoriesArray.push(
+          <BreachExposureCategory
+            key={item}
+            icon={<PhoneIcon alt="" width="13" height="13" />}
+            exposureCategoryLabel={l10n.getString("exposure-card-phone-number")}
+          />
+        );
+      } else if (item === "ip-addresses") {
+        exposureCategoriesArray.push(
+          <BreachExposureCategory
+            key={item}
+            icon={<PhoneIcon alt="" width="13" height="13" />}
```

Comment:
nit: I think we need a newer/different/better icon for IP Addresses than the PhoneIcon.